src/SkillData.lua

```diff
@@ -2340,10 +2504,14 @@ function TravelDictionary:CreateDictionaries()
         FR={ name="Retour à Bree",      desc="Permet de retourner",                label="Bree", zlabel="Porte Ouest",     zone="Bree", },
         RU={ name="Возвращение в Бри",  desc="Вы можете быстро перенестись в Бри", label="Бри",  zlabel="Западные ворота", zone="Бри", },
         map={MapType.ERIADOR, 525, 335},
-        acquire={ {
-            EN={deed="Enmity of the Wargs II",},
-            DE={deed="",},
-            FR={deed="",}} },
+        acquire={
+            {

```

Comment:
There is extra formatting on these Enmity of tags

Reply:
thanks, fixing